twoliter/embedded/rpm2img

```diff
@@ -186,61 +186,55 @@ INVENTORY_DATA="$(jq --raw-output . <<<"${installed_rpms[@]}")"
 # Sort by package name and add 'Content' as top-level.
 INVENTORY_DATA="$(jq --slurp 'sort_by(.Name)' <<<"${INVENTORY_DATA}" | jq '{"Content": .}')"
 
-# Get the core kit version and vendor from external kit metadata.
+# Iterate through all kits used to build this variant.
 EXTERNAL_KIT_METADATA_PATH="${EXTERNAL_KITS_PATH}/external-kit-metadata.json"
-CORE_KIT_VERSION=$(jq --raw-output '.kit[]|select(.name == "bottlerocket-core-kit")|.version' "${EXTERNAL_KIT_METADATA_PATH}")
-CORE_KIT_VENDOR=$(jq --raw-output '.kit[]|select(.name == "bottlerocket-core-kit")|.vendor' "${EXTERNAL_KIT_METADATA_PATH}")
-# Set the path inside the build container to the core kit RPMs and repo.
-CORE_KIT_PATH="${EXTERNAL_KITS_PATH}/${CORE_KIT_VENDOR}/bottlerocket-core-kit/${ARCH}"
-
-if [[ -n "${CORE_KIT_VERSION}" && -n "${CORE_KIT_VENDOR}" ]]; then
-  # Query the bottlerocket-core-kit repo for a single package's Release in order
-  # to extract the commit that built the core kit.
-  CORE_KIT_GIT_SHA="$(dnf --repofrompath \
-    core-kit,file://"${CORE_KIT_PATH}" \
-    --repo=core-kit repoquery \
-    --queryformat '%{Buildtime} %{Release}' 2>/dev/null | \
-    sort -k 1,2 | \
-    awk -F '.' 'END {print $--NF}')"
-  if [[ -z "${CORE_KIT_GIT_SHA}" ]]; then
-    echo "Could not find Git sha for bottlerocket-core-kit" >&2
-    exit 1
+mapfile -t kits < <(jq -r ".kit[].name" "${EXTERNAL_KIT_METADATA_PATH}")
+for kit in "${kits[@]}"; do
+  KIT_VERSION=$(jq --arg kit "${kit}" --raw-output '.kit[]|select(.name == $kit)|.version' "${EXTERNAL_KIT_METADATA_PATH}")
+  KIT_VENDOR=$(jq --arg kit "${kit}" --raw-output '.kit[]|select(.name == $kit)|.vendor' "${EXTERNAL_KIT_METADATA_PATH}")
```

Comment:
style nit: for variables declared and used within a loop, I like to use lowercase variables as a gesture at proper scoping.